modules/rosa-imds-machine-pools.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: CONCEPT
+[id="rosa-imds-machine-pools_{context}"]
+= Instance Metadata Service on machine pools
+
+Instance metadata allows you to configure and manage a running instance. Instance metadate also allows you to access user data that you need to launch your instance.
+
+IMDSv2 uses session-oriented requests. With session-oriented requests, you create a session token that defines the session duration, which can be a minimum of one second and a maximum of six hours. During the specified duration, you can use the same session token for subsequent requests. After the specified duration expires, you must create a new session token to use for future requests.
```

Comment:
Related to another comment here - some hint about composing "the requests" would be nice.

Reply:
Per other comment, the "hint" is the AWS documentation. Good enough, imho.